packages/Notification/Notification.md

```diff
@@ -68,7 +76,14 @@ Use the `warning` variant to provide feedback of a warning; users would still be
 
 ### Dismissible
 
-Use the `dismissible` prop to allow users to dismiss the `Notification` at any time. The `dismissible` prop must be used together with the `dismissibleA11yLabel` to remain accessible. Once dismissed, the `Notification` is removed from the DOM.
+Notifications can either be passive or require action from the user. If the information is passive then it should be dismissible as indicated by an interactive close button on the far right. If the Notification requires an action then make the option(s) clear with a Chevron Link or Paragraph link style.
+
+Use the dismissible prop to allow users to dismiss the Notification at any time. The `dismissible` prop must be used together with the `dismissibleA11yLabel` to remain accessible. Once dismissed, the Notification is removed from the DOM.
+
+Please note:
+
+- Branded, General and Success and Instructional variants can be dismissed
```

Comment:
Lets remove **"and Instructional"**, why are we capitalizing General, and Success?

Reply:
Instructional is a variant type though. It's not currently deprecated, should it be?

Reply:
My mistake.